File: Tarea_7_PY/Banco.py
class Banco:
    def __init__(self):
        self.cuentas = []

    def agregarCuenta(self, cuenta):
        self.cuentas.append(cuenta)

    def listarCuentas(self):
        return self.cuentas

    def buscarCuenta(self, id):
        for cuenta in self.cuentas:
            if cuenta.getId() == id:
                return cuenta
        return None

File: Tarea_7/src/Banco.java
import java.util.*;
public class Banco {
    List<CuentaBancaria> cuentas;

    public Banco() {
        cuentas = new ArrayList<>();
    }

    public void agregarCuenta(CuentaBancaria cuenta) {
        cuentas.add(cuenta);
    }

    public List<CuentaBancaria> listarCuentas() {
        return cuentas;
    }

    public CuentaBancaria buscarCuenta(int id) {
        for (CuentaBancaria cuenta : cuentas) {
            if (cuenta.getId() == id) {
                return cuenta;
            }
        }
        return null;
    }
}
